src/components/PhotoAnalysis/PhotoAnalysis.tsx

```diff
@@ -64,32 +93,39 @@ const Analysis: React.FC = () => {
         }
       })
     }
-    labelData = labelsArray
+
     const labelValues = labelsArray.map(label => {
       return label.name
     })
     labels = labelValues
   }
+
   const pageData = (
     <div>
-      <div className="file is-flex-direction-column is-justify-content-center is-align-items-center">
-        <label className="file-label">
-          <input
-            className="file-input"
-            type="file"
-            onChange={identifyImageLabels}
-          ></input>
-          <span className="file-cta">
-            <span className="file-label">Choose a file…</span>
-          </span>
-        </label>
-      </div>
-      <br />
-      <img src={imageSrc} alt="user's uploaded file" />
-      {processRekognitionLabels(rekognitionResponse as IdentifyLabelsOutput)}
-      <p>{labels.join(", ")}</p>
+      {imageSrc !== undefined ? (
+        <div className="card-image">
+          <img src={imageSrc} alt="Uploaded preview" />
+        </div>
+      ) : null}
```

Comment:
The image needs to be displayed inside the `card-content` (or between the _take or upload photo_ text and the actual upload button)